src/pudl/transform/ferc1.py

```diff
@@ -829,17 +829,28 @@ def transform_main(self, df: pd.DataFrame) -> pd.DataFrame:
             self.normalize_strings(df)
             .pipe(self.categorize_strings)
             .pipe(self.convert_units)
+            .pipe(self.strip_non_numeric_values)
             .pipe(self.nullify_outliers)
             .pipe(self.drop_invalid_rows)
         )
-        df["project_num"] = self.strip_col_of_non_ints(df["project_num"])
         return df
 
-    def strip_col_of_non_ints(self, col: pd.Series) -> pd.DataFrame:
-        """Strip the column of any non-integer values."""
-        return pd.to_numeric(
-            col,
-            errors="coerce",
+
+class PlantsPumpedStorageFerc1TableTransformer(Ferc1AbstractTableTransformer):
+    """Transformer class for ``plants_pumped_storage_ferc1`` table."""
+
+    table_id: Ferc1TableId = Ferc1TableId.PLANTS_PUMPED_STORAGE_FERC1
+
+    @cache_df(key="main")
+    def transform_main(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Table-specific transforms for the ``plants_pumped_storage_ferc1`` table."""
+        return (
+            self.normalize_strings(df)
+            .pipe(self.categorize_strings)
+            .pipe(self.convert_units)
+            .pipe(self.strip_non_numeric_values)
+            .pipe(self.nullify_outliers)
+            .pipe(self.drop_invalid_rows)
```

Comment:
I have very little familiarity with these tables, so sorry if this is naive. I'm trying to think of ways to justify moving these cleaning functions into `Ferc1AbstractTableTransformer`.

Similar events happen in the steam transform it seems like, but in a different order:
```
self.normalize_strings(df)
.pipe(self.nullify_outliers)
.pipe(self.categorize_strings)
.pipe(self.convert_units)
.pipe(self.drop_invalid_rows)
```
Do these functions need to happen in this order or can they happen in the same order as in the pumped storage and hydro transform? For the sake of generalizing, if `strip_non_numeric_values` was included in this "general cleaning functions" block of code, could you apply it to the steam table and just not set any `strip_non_numeric_values` columns in the steam table params? 

If the same cleaning functions are applied for steam, hydro, and pumped storage, then I think you could justify moving the above code block of "generic" cleaning sequence to a function in `Ferc1AbstractTableTransformer` and all the transforms would read a little cleaner.

Reply:
I think this is a pretty good idea! this does seem to be the pattern. I was holding off until we could actually see a pattern but after two tables of basically identical `transform_main`'s IT IS TIME

Reply:
that worked great!